src/types/index.ts

```diff
@@ -207,15 +207,15 @@ export interface TokenDocument {
  * Type of Authorization Request
  */
 export enum AuthorizationType {
-  AttestPrimaryKeyRotation = 'AttestPrimaryKeyRotation',
-  RotatePrimaryKey = 'RotatePrimaryKey',
+  AttestPrimaryAccountRotation = 'AttestPrimaryAccountRotation',
+  RotatePrimaryAccount = 'RotatePrimaryAccount',
   TransferTicker = 'TransferTicker',
   AddMultiSigSigner = 'AddMultiSigSigner',
   TransferAssetOwnership = 'TransferAssetOwnership',
   JoinIdentity = 'JoinIdentity',
   PortfolioCustody = 'PortfolioCustody',
   BecomeAgent = 'BecomeAgent',
-  AddRelayerPayingKey = 'AddRelayerPayingKey',
+  AddRelayerPayingAccount = 'AddRelayerPayingAccount',
```

Comment:
These also mirror chain auth types (and we actually send the values to the chain when creating auths) so they should be left as is. We might consider having Account on the left side and key on the right, but I think it might do more harm than good in terms of readability

The corresponding authorization assertion functions (together with their comments and test descriptions) and "FooAuthorizationData" types should also be renamed back

Reply:
Thanks I wasn't sure. I almost left them but decided to change them near the end.

Wil revert back.